iqm5q.py

```diff
@@ -73,7 +79,7 @@ def create(runcard_path=RUNCARD):
         descriptor,
         use_emulation=False,
         time_of_flight=75,
-        smearing=50,  # time_of_flight=280, smearing=100
+        smearing=50,
     )
 
     # Create channel objects and map controllers
```

Comment:
Sweep labels for readout and feedback channels:

 # feedback
    channels |= Channel(
        "L2-7", port=controller[("device_shfqc", "[QACHANNELS/0/INPUT]")]
    )
    # readout
    channels |= Channel(
        "L3-31", port=controller[("device_shfqc", "[QACHANNELS/0/OUTPUT]")]
    )